packages/trading-widget/src/core-kit/hooks/trading/withdraw-v2/init-step/use-init-withdraw-transaction-arguments.ts

```diff
@@ -0,0 +1,51 @@
+import BigNumber from 'bignumber.js'
+import { useMemo } from 'react'
+
+import { DEFAULT_PRECISION } from 'core-kit/const'
+import {
+  useSendTokenInput,
+  useTradingPanelPoolConfig,
+} from 'core-kit/hooks/state'
+import { useIsMultiAssetWithdraw } from 'core-kit/hooks/trading/withdraw-v2/init-step/use-is-multi-asset-withdraw'
+import { useAppliedWithdrawSlippage } from 'core-kit/hooks/trading/withdraw-v2/use-applied-withdraw-slippage'
+import { useDebounce } from 'core-kit/hooks/utils'
+import { getSlippageToleranceForWithdrawSafe } from 'core-kit/utils'
+
+interface UseInitWithdrawTransactionArguments {
+  debounceTime?: number
+}
+
+export const useInitWithdrawTransactionArguments = ({
+  debounceTime,
+}: UseInitWithdrawTransactionArguments = {}) => {
```

Comment:
Can we define return type to avoid casting while using args?

Reply:
got rid of types casting